percona/controller/pgcluster/extensions.go

```diff
@@ -50,6 +50,13 @@ func ExtensionInstallerContainer(postgresVersion int, spec *v2.ExtensionsSpec, e
 		Image:           spec.Image,
 		ImagePullPolicy: spec.ImagePullPolicy,
 		Command:         []string{"/usr/local/bin/install-extensions.sh"},
+		SecurityContext: &corev1.SecurityContext{
+			RunAsUser: func() *int64 {
+				// postgres user
+				var uid int64 = 26
```

Comment:
@egegunes I hope it will work with OpenShift

Reply:
I think it'll. Our postgres image is also running with this user id